src/ui/pages/Backup/RecoveryPhrase.tsx

```diff
@@ -141,9 +148,11 @@ export function Backup({
                 >
                   Verify Backup
                 </Button>
-                <Button kind="ghost" onClick={onSkip}>
-                  Do it Later
-                </Button>
+                {context.name === 'onboarding' ? (
+                  <Button kind="ghost" onClick={onSkip}>
+                    Do it Later
+                  </Button>
+                ) : null}
```

Comment:
We want to show "Do it Later" only during onboarding. Otherwise, it would be odd to display this button right after the user clicked "Backup," as it contradicts their current intention